packages/browser-sdk/src/client.ts

```diff
@@ -148,6 +148,18 @@ export class BucketClient {
 
     await this.featuresClient.initialize();
 
+    if (this.context.user) {
+      this.user(this.context.user).catch((e) => {
```

Comment:
I wonder why my lsp complains about the unawaited promises and yours doesnt. I usually add the special void keyword in front to make typescript understand I don't want to await.

Reply:
agreed, pretty weird. I've learned from your comments to always add the `void`. These two are awaited though :)

Reply:
The catch call returns another promise though

Reply:
`catch` is enough AFAICT? : https://typescript-eslint.io/rules/no-floating-promises/

Reply:
Interesting. I think I've configured my TS LSP to be stricter and that's why it won't let me be in such cases. But i agree, it's enough.